androidApp/src/main/java/com/example/tasky/android/agenda/components/DetailsReminderSection.kt

```diff
@@ -39,44 +53,65 @@ fun DetailsReminderSection(
     onReminderSelect: (ReminderType) -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    Row(modifier = modifier, verticalAlignment = Alignment.CenterVertically) {
-        Box(
-            modifier =
-                Modifier
-                    .size(30.dp)
-                    .background(Light2),
-        ) {
-            Icon(
-                Icons.Outlined.Notifications,
-                contentDescription = null,
-                tint = Gray,
+    var isMenuOpen by remember {
+        mutableStateOf(false)
+    }
```

Comment:
Here, your dropdown menu will not remain open when a configuration change (device orientation or change to dark mode) occurs because the `Activity` is recreated. Thus, your `isMenuOpen` state will be reset to false.

I'd strongly recommend to put all UI states in a ViewModel. With `rememberSaveable` you could work around that, but then your UI will have too many responsibilities, since it also needs to decide how the state is updated.